msteams-platform/toolkit/publish.md

```diff
@@ -72,20 +72,16 @@ When the app is ready for use in production, you can submit the app using the Te
 The following steps help you to publish the app from Teams Toolkit:
 
 1. You can publish your Teams app in one of the following ways:
-     * Select **Publish to Teams** under **DEPLOYMENT**.
+     * Select **Publish** under **LIFECYCLE**.
      * Select **View** > **Command Palette...** > **Teams: Publish to Teams**.
```

Comment:
The command in command palette has been changed from "Teams: Publish to Teams" to "Teams: Publish".

Please confirm @jidddddd

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/71362691/ee4128b3-e029-4e44-b31f-3b126b0294d2)

Reply:
Updated/Resolved.